docs/profiles.md

```diff
@@ -31,12 +31,12 @@ Attribute                     | Type     | Notes
 ------------------------------|----------|----------------------------------------------------------------------------
 weight_name                   | String   | Name used in output for the routing weight property (default 'duration')
 weight_precision              | Unsigned | Decimal precision of edge weights (default 1)
-left_hand_driving             | Boolean  | Are vehicles assumed to drive on the left? (used in guidance)
-use_turn_restrictions         | Boolean  | Are turn instructions followed?
-continue_straight_at_waypoint | Boolean  | Must the route continue straight on at a via point, or are U-turns allowed?
+left_hand_driving             | Boolean  | Are vehicles assumed to drive on the left? (used in guidance, default false)
```

Comment:
Good catch, these should be enclosed too. It's just easier to differentiate what a value and what a name is with `code` tags. 😉